extensions/splix/package.json

```diff
@@ -13,26 +13,23 @@
   "commands": [
     {
       "name": "search",
-      "title": "Search Note",
+      "title": "Search Notes",
       "subtitle": "Splix",
-      "description": "Search for a note in Splix",
+      "description": "Search for notes in Splix",
       "mode": "view"
     },
     {
-      "name": "add-note",
-      "title": "Add Note",
+      "name": "new-note",
```

Comment:
Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

Reply:
Fixed